import java.util.*;
import java.io.FileNotFoundException;

 public class prac4{
void method1(){
  try{}
  catch(Error e){}
}

void method2(){
  try{}
   catch(Exception e){}
}

void method3(){
  try{}
  catch(Throwable e){}
}

void method4(){
   try{}
 catch(RuntimeException e){}
}
/* It is checked type of the exception 
  
void method5(){
   try{}
    catch(FileNotFoundException e){}
}*/

  public static void main(String args[])throws FileNotFoundException{

  
  



}
}
 
